src/components/implementation/tests/micro_booter/micro_booter.h

```diff
@@ -9,49 +9,80 @@
 
 #undef assert
 /* On assert, immediately switch to the "exit" thread */
-#define assert(node)                                          \
-	do {                                                  \
-		if (unlikely(!(node))) {                      \
-			debug_print("assert error in @ ");    \
-			cos_thd_switch(termthd[cos_cpuid()]); \
-		}                                             \
-	} while (0)
+#define assert(node)                                            \
```

Comment:
High level-comment: I think that we need to more intentionally design which files contain which functionality.  A single driver .c file, with a c file for each "cluster" of tests, with a single .h file to tie them together with shared functionality seems right to me.